definitions/variable/technology/power-plant.yaml

```diff
@@ -125,86 +127,63 @@
       from {Electricity Input}
     unit: MW/h
     skip-region-aggregation: true
-- Maximum Storage|Electricity|Hydro|Reservoir:
-    description: Maximum volume of a reservoir expressed in energy
-    unit: GWh
-- Minimum Storage|Electricity|Hydro|Reservoir:
-    description: Minimum volume of a reservoir expressed in energy
-    unit: GWh
-- Pumping Efficiency|Electricity|Hydro|Reservoir:
-    description: Efficiency of pumping for a hydro reservoir power plant
+- Maximum Storage|Electricity|{Electricity Storage Type}:
+    description: Maximum volume of a {Electricity Storage Type} unit expressed in energy
+    unit:
+    - MWh
+    - GWh
+- Minimum Storage|Electricity|{Electricity Storage Type}:
+    description: Minimum volume of a {Electricity Storage Type} unit expressed in energy
+    unit:
+    - MWh
+    - GWh
+- Charging Efficiency|Electricity|{Electricity Storage Type}:
+    description: Efficiency of charging energy for a {Electricity Storage Type}
     unit: '%'
     skip-region-aggregation: true
-- Turbine Efficiency|Electricity|Hydro|Reservoir:
-    description: Efficiency of turbining for a hydro reservoir power plant
+- Discharging Efficiency|Electricity|{Electricity Storage Type}:
+    description: Efficiency of discharging for a {Electricity Storage Type}
     unit: '%'
     skip-region-aggregation: true
-- Inflows|Electricity|Hydro|Reservoir:
-    description: Inflows into a reservoir expressed in energy
+- Inflows|Electricity|{Electricity Storage Type}:
+    description: Inflows into a storage expressed in energy
+    unit: GWh
+- Outflows|Electricity|{Electricity Storage Type}:
+    description: Outflows from a storage expressed in energy to cover a demand outside of the electricity system
     unit: GWh
-- Inflows|Electricity|Hydro|Pumped Storage:
-    description: Inflows into a reservoir expressed in energy
+- Initial Storage Level|Electricity|{Electricity Storage Type}:
```

Comment:
I suggest to remove that variable as it is redundant with 'storage' at a given date, and is not accurate enough